docs/guide/debugging.md

```diff
@@ -4,6 +4,18 @@ title: Debugging | Guide
 
 # Debugging
 
+## Terminal
+
+To debug a test file in your shell, add a `debugger` statement anywhere in your code, and then run `ndb`:
+
+```sh
+# install ndb globally
+npm install -g ndb
```

Comment:
Can we add a reference to ndb docs?

Reply:
ndb doesn't have any docs, but we can link to the github repo. It has a helpful readme.